habitat-lab/habitat/tasks/nav/nav.py

```diff
@@ -1195,10 +1243,9 @@ def step(
         self,
         *args: Any,
         task: EmbodiedTask,
-        linear_velocity: float,
-        angular_velocity: float,
+        linear_velocity: float = None,
```

Comment:
I dont think we will want to have a case where None is passed to the agent. Lets remove it